src/sentry/static/sentry/app/components/acl/access.tsx

```diff
@@ -22,11 +23,6 @@ export type ChildRenderProps = {
 
 type ChildFunction = (props: ChildRenderProps) => React.ReactNode;
 
-// Type guard for render func.
-function isRenderFunc(func: React.ReactNode | Function): func is ChildFunction {
```

Comment:
We probably have a few of these floating around

Reply:
I'll try to find them all. I can only assume they have different names 😬